R/layer_yeo_johnson.R

```diff
@@ -0,0 +1,249 @@
+#' Unormalizing transformation
+#'
+#' Will undo a step_epi_YeoJohnson transformation.
+#'
+#' @param frosting a `frosting` postprocessor. The layer will be added to the
+#'   sequence of operations for this frosting.
+#' @param lambdas Internal. A data frame of lambda values to be used for
+#'   inverting the transformation.
+#' @param ... One or more selector functions to scale variables
+#'   for this step. See [recipes::selections()] for more details.
+#' @param by A (possibly named) character vector of variables to join by.
```

Comment:
Is the intention a `join_by()` object? Even if not, it would be helpful to expand here along the lines of `?dplyr::left_join`.

Reply:
Alternatively, `@inheritParams step_population_scaling`. Actually, that should cover most of the examples.